vwr/driver.py

```diff
@@ -83,10 +86,14 @@ def turn_off(self):
 
     def get(self):
         """Gets the setpoint and internal temperature."""
-        return {'setpoint': self.get_setpoint(),
-                'actual': self.get_internal_temperature(),
-                'pump': self.get_pump_speed(),
-                'on': self.get_operating_status()}
+        response = {'setpoint': self.get_setpoint(),
+                    'actual': self.get_internal_temperature(),
+                    'pump': self.get_pump_speed(),
+                    'on': self.get_operating_status()}
+        if self.connected:
+            return response
+        else:
+            return {'connected': False}
```

Comment:
This should remove a lot of the complexity from the controller. I debated changing the `set` methods to respond with something like this but I think returning True or False like they did before is enough to handle in the controller. I also noticed that currently the filter controller doesn't do anything with the value returned by the drivers `set` methods. So I'll add something for that too.